checkstyle/suppressions.xml

```diff
@@ -25,9 +25,9 @@
               files="(S3SinkConfig).java"/>
 
     <suppress checks="(ClassFanOutComplexity)"
-              files="(S3SinkConfig|S3SinkTaskTest|IntegrationTest).java"/>
+              files="(S3SinkTask|S3SinkConfig|S3SinkTaskTest|IntegrationTest).java"/>
```

Comment:
This checks are basically useless nowadays since we just add exceptions there. Either we should drop them or refactor the classes (maybe except for tests).